src/__screenshot_tests__/logo-screenshot-test.tsx

```diff
@@ -0,0 +1,45 @@
+import {openStoryPage} from '../test-utils';
+
+test.each`
+    isInverse | isDarkMode | logoType       | size
+    ${false}  | ${false}   | ${'imagotype'} | ${48}
+    ${false}  | ${false}   | ${'vertical'}  | ${48}
+    ${false}  | ${false}   | ${'isotipo'}   | ${48}
+    ${false}  | ${true}    | ${'imagotype'} | ${48}
+    ${false}  | ${true}    | ${'vertical'}  | ${48}
+    ${false}  | ${true}    | ${'isotipo'}   | ${48}
+    ${true}   | ${false}   | ${'imagotype'} | ${48}
+    ${true}   | ${false}   | ${'vertical'}  | ${48}
+    ${true}   | ${false}   | ${'isotipo'}   | ${48}
+    ${true}   | ${true}    | ${'imagotype'} | ${48}
+    ${true}   | ${true}    | ${'vertical'}  | ${48}
+    ${true}   | ${true}    | ${'isotipo'}   | ${48}
+`(
+    'Logo. isInverse={$isInverse} isDarkMode={$isDarkMode} logoType={$logoType}',
+    async ({isInverse, isDarkMode, logoType, size}) => {
+        await openStoryPage({
+            id: 'components-logo--default',
+            device: 'DESKTOP',
```

Comment:
depending on if we want some size defaults according to the device, we can add more